packages/hub/src/scope.ts

```diff
@@ -217,6 +226,19 @@ export class Scope implements ScopeInterface {
     return this._span;
   }
 
+  /**
+   * @inheritDoc
+   */
+  public getTransaction(): Transaction | undefined {
```

Comment:
Just wanted to note that in terms of Unified API, this is going to be the first public and documented getter on the `Scope`. Up until know the scope is a write-only concept, it can be mutated but not inspected.

(The other getter, `getSpan` is marked as `@hidden` and is also part of adding Tracing to the API).